sootup.core/src/main/java/sootup/core/graph/BackwardsStmtGraph.java

```diff
@@ -35,12 +34,11 @@ public BackwardsStmtGraph(@Nonnull StmtGraph<?> stmtGraph) {
 
   @Override
   public Stmt getStartingStmt() {
-    throw new UnsupportedOperationException();
-  }
-
-  @Nonnull
-  public List<Stmt> getStartingStmts() {
-    return backingGraph.getTails();
+    List<Stmt> tails = backingGraph.getTails();
+    if (tails.isEmpty()) {
+      return null;
+    }
+    return tails.get(0);
```

Comment:
I think we should keep it with the UnsupportedException instead of arbitrarily choosing one Tail.

Reply:
Got it